src/sql/workbench/services/connection/browser/connectionWidget.ts

```diff
@@ -646,6 +646,9 @@ export class ConnectionWidget extends lifecycle.Disposable {
 				// Populate tenants (select first by default for initialization of tenant dialog)
 				await this.onAzureAccountSelected(true);
 				this._azureTenantDropdown.enable();
+				// Populate username as 'email' of selected azure account in dropdown (Ask from MySQL)
```

Comment:
Generally I don't think putting information about who requested a feature is a useful thing to have in a comment. Maybe instead say why this is a useful thing to have?

Reply:
```suggestion
				// Populate username as 'email' of selected azure account in dropdown, as username is required, and email of Azure account selected applies as username in most cases.
```